src/test/java/com/microsoft/azure/functions/worker/broker/tests/FunctionDefinitionTest.java

```diff
@@ -11,65 +12,65 @@
 
 
 public class FunctionDefinitionTest {
```

Comment:
I remember I already updated this tests, why we update again here?

Reply:
Ran into some conflicts when I rebased with dev. This change got overwritten

Reply:
Okay, once we are sure that no other changes (changes except rename and tests) from the initial PR I am good with this PR.

Reply:
Yes, all changes to go with recent host changes are done